documentation/modules/upgrading/proc-downgrade-brokers-older-kafka.adoc

```diff
@@ -75,3 +76,16 @@ Reconciliation #_NUM_(watch) Kafka(_NAMESPACE_/_NAME_): Kafka version downgrade
 . Downgrade all client applications (consumers) to use the previous version of the client binaries.
 +
 The Kafka cluster and clients are now using the previous Kafka version.
+
+. If you are reverting back to a version of Strimzi that uses ZooKeeper for the storage of topic metadata, delete the internal topic store topics from the Kafka cluster.
++
+For example, where _my-cluster-kafka-0_ is the name of the broker pod:
++
+[source,shell]
+----
+kubectl exec -ti my-cluster-kafka-0 -- ./bin/kafka-topics.sh --bootstrap-server localhost:9092 --topic __strimzi-topic-operator-kstreams-topic-store-changelog --delete && ./bin/kafka-topics.sh --bootstrap-server localhost:9092 --topic __strimzi_store_topic --delete
+----
```

Comment:
Right place? Do we want to say _Optional_ or _earlier than 0.22_?

Reply:
To make it future proof you should probably say something like _when downgrading to operator version earlier then 0.22, you should ..._

Reply:
Added